### DISCUSSION TITLE
Would Gcam Services Provider work for Google Maps as well

### DISCUSSION THREAD

**Discussion by ghost** [lukaspieper/Gcam-Services-Provider#41] [General]
Hi,

Would this app enable Google Maps to work and enable push notifications for all apps. 
Thanks.

**Comment by lukaspieper** [comment]
No it doesn't.

The very fact that the app does not have these functions is more or less the reason for using it. As far as I know, there are no side effects in other apps when using Gcam services providers. The aim of this app is exclusively to make Gcam work.

**Comment by innir** [comment]
Google Maps should actually work without it: https://plexus.techlore.tech/applications/google-maps

**Reply by lukaspieper** [reply]
Google Maps' navigation works without any problems on devices without Play Services, although a warning message is displayed that can be easily closed. The link from @innir is therefore correct (you need to look at the left side that says "de-googled").

Furthermore, microG does not force you to log in with a Google account, many functions also work without an account.

Just a personal note on the side, I think you focus far too much on the phone number. Google has much more advanced methods of matching profiles than using the phone number. If you're so worried, which I don't think is a bad thing, maybe you shouldn't use Google Maps.

**Reply by lukaspieper** [reply]
The following reflects only my opinion/knowledge. The whole topic is very debatable.

Regarding your disconnecting the internet, two thoughts. First, apps are perfectly capable of caching everything on the device and sending it as soon as the connection is back up. Google even provides a pretty easy-to-use library for developers to do this. Second, you might make yourself conspicuous, your behaviour differs from the masses.
From my point of view, this doesn't even matter as long as you have the original proprietary Google services on your device (so you don't need my app anyway).

[Compared to microG](https://github.com/microg/GmsCore/wiki/Implementation-Status), the Play services you use include all analytics and advertising features. So, I don't really understand you yet. You said that you don't want microG because you can log in and instead use the original services? Or are we talking about multiple devices?

Switching from the original Play services to microG should get you much further than temporarily disconnecting apps from the internet. Disconnecting apps completely from the internet makes sense because then it doesn't matter what they cache.

Regarding the firewall: I'm not interested in such a solution because it would be a permanent game of cat and mouse. The person who maintains such a list would have to invest an enormous amount of time and no one can tell you whether the list would be complete. 

tl;dr: Your current approaches only fight the symptoms, not the causes. No offence intended, everyone starts somewhere.
